web/src/app/components/header/header.component.html

```diff
@@ -54,14 +54,16 @@
         <button
           mat-flat-button
           class="finish-edit-button"
-          [disabled]="!isDraftSurveyDirtyAndValid()"
+          [disabled]="publishingChanges || !isDraftSurveyDirtyAndValid()"
           (click)="onFinishEditSurveyClick()"
         >
           Publish changes
           <mat-icon *ngIf="publishingChanges">
             <mat-spinner diameter="18"></mat-spinner>
           </mat-icon>
         </button>
+
+        <div class="header-divider"></div>
```

Comment:
It looks like the header divider styles were removed in `_survey-header.component-theme.scss`. Is this supposed to still be here?

Reply:
good point. survey header is just a customisation of <ground-header>.

the best thing to do is move survey-header component out from main-page and merge it with ground-header. I'll file a code health issue.